src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -411,7 +411,7 @@ module.exports.commands = [{
   validateLogConsole: function (position) {
       const self = this;
       const browser = this.api;
-      const aXpath = '(//table[@class="highlight"]//tr/td/a)[' + position + ']';
+      const aXpath = '(//pre//p/div/a)[' + position + ']';
```

Comment:
hmm I understand what you mean, but it is the first time that we use `<pre/>` Anyway will make it specific for the log console DOM tree